<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-EGH5GRK4" ignored="0" name="新增项目接口" tests="1" failures="1" timestamp="23 八月 2020 04:14:21 GMT" time="0.182" errors="0">
  <testcase name="test" time="0.125" classname="com.api.cases.AddCase">
    <failure type="java.lang.AssertionError" message="expected [passed] but found [failed]">
      <![CDATA[java.lang.AssertionError: expected [passed] but found [failed]
at com.api.cases.AddCase.test(AddCase.java:44)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
</testsuite> <!-- 新增项目接口 -->
